### PR TITLE
Added Goblin icon to the Index footprint library, and placed it on the bottom of Mobo-rev03

### PR DESCRIPTION
**Pull request by sphawes**
like it says in the title, added the goblin to the board!

### CODE REVIEW

**Comment by daveismith** [comment]
@sphawes - that goblin looks pretty comfortable hanging out there in the corner :)

mobo-bottom

File: lfs.sh [hunk 1]
```diff
@@ -69,5 +69,5 @@ esac
 # Lock a file list
 for f in $FILES
 do
-  git lfs $1 $3 $f
+  git lfs $1 $3 $f --remote upstream
```
Comment (daveismith) [inline lfs.sh:72]: I don't have the remote titled upstream and generally just use "--remote <remote_name>" as an argument to the script.

Comment (sphawes) [reply lfs.sh:72]: oh, that's much easier! is that what the $3 allows for? I'll revert this change

Comment (daveismith) [reply lfs.sh:72]: I believe that's what the $3 was for. The only strange part is needing to put it in quotes. There's probably a way to handle it with the `$@` variable and removing a few items from the top.

**Comment by G-Pereira** [comment]
Cool :)

**Pull request merged**